package-lock.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "cardservernode",
+  "name": "exceedgameserver",
```

Comment:
was there any particular reason you changed this? just curious

Reply:
I never looked at this file, whatever this change is was automated

Reply:
Not sure what to do about this, pls advise

Reply:
I think this is probably fine. I suspect it is just based on the git name, which I changed.